### PR TITLE
Montessori training model rspec

### PR DESCRIPTION
**Pull request by dovyblue**

### CODE REVIEW

File: spec/factories/references.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'faker'
```
Comment (trevordjones) [inline spec/factories/references.rb]: You don't need these `require 'faker'` lines. You can remove from this and the other files

File: spec/models/reference_spec.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Reference, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Reference do
+  it 'is valid with name address position' do
+    expect(build(:reference)).to be_valid
+  end
```
Comment (trevordjones) [inline spec/models/reference_spec.rb]: Could you do some invalid tests here as well? Like you did with `MontessoriTrainingSpec`?

Comment (dovyblue) [inline spec/models/reference_spec.rb]: It doesn't have any validations, should I add?

File: spec/factories/references.rb (hunk 1 above)
Comment (dovyblue) [inline spec/factories/references.rb]: Ok

File: spec/models/reference_spec.rb (hunk 2 above)
Comment (trevordjones) [inline spec/models/reference_spec.rb]: Oh, if it doesn't have any validations then we don't need to test at all. Looks like it has a `belongs_to` association so maybe test for that instead.

**Pull request closed without merging**